lib/perl/Genome/InstrumentData/Command/Import/WorkFlow/FastqsToBam.pm

```diff
@@ -127,29 +86,25 @@ sub _fastqs_to_bam {
 
     my @fastqs = $self->fastq_paths;
     $self->debug_message("Fastq 1: $fastqs[0]");
+
     my $output_bam_path = $self->output_path;
-    my %fastq_to_sam_params = (
-        fastq => $fastqs[0],
-        output => $output_bam_path,
-        quality_format => 'Standard',
-        sample_name => $self->library->sample->name,
-        library_name => $self->library->name,
-        read_group_name => UR::Object::Type->autogenerate_new_object_id_uuid,
-        use_version => '1.113',
-    );
+
+    my $cmd = '/gapp/x64linux/opt/java/jre/jre1.8.0_31/bin/java -Xmx16g -jar /gscmnt/gc2560/core/software/picard/2.15.0/picard.jar FastqToSam O='. $output_bam_path .' SM='. $self->library->sample->name .' LB='. $self->library->name .' RG='. UR::Object::Type->autogenerate_new_object_id_uuid .' F1='. $fastqs[0];
```

Comment:
Do we care about continuing to support `gmt picard` in general?  `GMT::Picard::Base` supports specifying a Java interpreter and, if told about this new Picard version, would probably work to continue being a viable wrapper for `FastqToSam`.  We'd have to update this particular GMT to the `Base` version, though.

Reply:
I envision the tool tree as-is going away and being replaced by a dockerized version of tools/images. The transition of these changes can take considerable time so maintaining the picard tool tree is still required.

However, I think the other issue was with the JAR files. After 1.123 I think Picard switched from tool specific JARs to a single `picard.jar` where you define the sub-command passed as an arg to the main JAR.  It was always my understanding this was the limitation to updating Picard in the GMT tree. We'd have to rewrite how the JAR and subcommand names are defined.

Reply:
We do at least try to handle the single-JAR case (as seen [here](https://github.com/genome/genome/blob/68b99044894435f7d08d9621e4d8d606f23201d1/lib/perl/Genome/Model/Tools/Picard.pm#L246)).  Version 1.130 is installed on our systems via a package as well.  The current code expects any newer versions to also be installed via Debian package, so we'd need to do something special if we're going to put them in an allocation instead.

Reply:
I don't view debian packaging as a reasonable solution. All effort for future development should be directed toward supporting Docker images instead.

Reply:
In this case "something special" might mean adding to [this array](https://github.com/genome/genome/blob/68b99044894435f7d08d9621e4d8d606f23201d1/lib/perl/Genome/Model/Tools/Picard.pm#L147) or having a standard way to look up allocations for tool versions.  (The hacky version of the latter might be to have the tool be the `owner_class_name` and the version be the `owner_id`, but we could do better :smile:)

If we had `docker run` I could imagine all sorts of ways to update the GMS!